e3sm_to_cmip/_logger.py

```diff
@@ -11,11 +12,15 @@ def _setup_root_logger() -> str:  # pragma: no cover
     The logger module will write to a log file and stream the console
     simultaneously.
 
+    The log files are saved in a `/logs` directory relative to where
+    `e3sm_to_cmip` is executed.
+
     Returns
     -------
     str
         The name of the logfile.
     """
+    os.makedirs("logs", exist_ok=True)
```

Comment:
Hey @chengzhuzhang, here's the one line fix for #187. Thanks for catching this bug.